README.md

```diff
@@ -1,7 +1,11 @@
 # Palindrom
 <p align="center">
   <img alt="Palindrom Logo" src="https://cloud.githubusercontent.com/assets/17054134/23406359/e2690c08-fdbe-11e6-8880-c23dc771f899.png">
-</p> 
+</p>
```

Comment:
"Binds client side.." is tottaly non-accessible. Personally, I dont find anything fancy in this font, I think it could go as well in regular markup. Then it will not only become readable for all readers but also for bots/sites that uses github readmes for example as package descriptions.